pkg/contexts/ocm/accessmethods/wget/method.go

```diff
@@ -0,0 +1,151 @@
+package wget
+
+import (
+	"fmt"
+	"io"
+	"sync"
+
+	"github.com/open-component-model/ocm/pkg/blobaccess"
+	"github.com/open-component-model/ocm/pkg/blobaccess/wget"
+	"github.com/open-component-model/ocm/pkg/contexts/credentials"
+	"github.com/open-component-model/ocm/pkg/contexts/credentials/builtin/wget/identity"
+	"github.com/open-component-model/ocm/pkg/contexts/ocm/cpi/accspeccpi"
+	"github.com/open-component-model/ocm/pkg/mime"
+	"github.com/open-component-model/ocm/pkg/runtime"
+)
+
+// Type is the access type for a blob on an http server .
+const (
+	Type   = "wget"
+	TypeV1 = Type + runtime.VersionSeparator + "v1"
+)
+
+func init() {
+	accspeccpi.RegisterAccessType(accspeccpi.NewAccessSpecType[*AccessSpec](Type))
+	accspeccpi.RegisterAccessType(accspeccpi.NewAccessSpecType[*AccessSpec](TypeV1))
+}
+
+func Is(spec accspeccpi.AccessSpec) bool {
+	return spec != nil && spec.GetKind() == Type
+}
+
+// New creates a new Helm Chart accessor for helm repositories.
+func New(url, mime string) *AccessSpec {
+	return &AccessSpec{
+		ObjectVersionedType: runtime.NewVersionedTypedObject(Type),
```

Comment:
I don't fully understand the `NewVersionedTypedObject`, but shouldn't it be used with `TypeV1`? 

```suggestion
		ObjectVersionedType: runtime.NewVersionedTypedObject(TypeV1),
```

Reply:
Functionally, it should not make a difference. Besides, it seems like the conventions for the existing access methods is using the `Type` in the `New` method.